src/test/java/mshantadze/SearchTest.java

```diff
@@ -16,20 +20,155 @@ public class SearchTest extends BaseTest {
 
     @Test
     public void searchTest() {
-        HomePage homePage = new HomePage(driver);
-        homePage.verifyThatPageIsLoaded();
-        homePage.getSearchSection().typeInSearchField("Ball");
-        homePage.getSearchSection().clickSearchButton();
-        ProductListingPage plp = new ProductListingPage(driver);
-        List<WebElement> productNameLbls = plp.getProductNames();
-        for (WebElement productNameLbl : productNameLbls) {
-            String productName = productNameLbl.getText().toLowerCase();
-            LOGGER.info(productName);
-
-            Assert.assertTrue(
-                    productName.contains("ball"),
-                    "Expected element to contain word 'ball' but result was" + productName
-            );
+        WebDriver driver = new ChromeDriver();
+        try {
```

Comment:
Don't use try catch block on test layer, try to move error handling to pages/service layer